pyxem/__init__.py

```diff
@@ -47,11 +47,18 @@
 from .signals.electron_diffraction1d import LazyElectronDiffraction1D
 from .signals.electron_diffraction2d import LazyElectronDiffraction2D
 
-from .signals.vdf_image import VDFImage
 from .signals.crystallographic_map import CrystallographicMap
+
+from .generators.red_intensity_generator import ReducedIntensityGenerator
+from .generators.pdf_generator import PDFGenerator
+from .generators.variance_generator import VarianceGenerator
+from .signals.diffraction_variance1d import DiffractionVariance1D
+
 from .signals.diffraction_vectors import DiffractionVectors
 from .signals.indexation_results import TemplateMatchingResults
 from .signals.vdf_image import VDFImage
+from .signals.pdf1d import PDF1D
```

Comment:
I think this would be better as PairDistributionFunction1D

Reply:
Changed.